web/interfaces.js

```diff
@@ -165,7 +165,8 @@ class IPDFAnnotationLayerFactory {
   /**
    * @param {HTMLDivElement} pageDiv
    * @param {PDFPage} pdfPage
-   * @param {AnnotationStorage} [annotationStorage]
+   * @property {AnnotationStorage} [annotationStorage] - Storage for annotation
```

Comment:
This should be `@param` given that it's a function parameter and not a type property.

Reply:
Sorry, too much copy-and-paste; fixed now!